package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "app",
+  "private": "true",
+  "dependencies": {
+    "@babel/preset-react": "^7.23.3",
+    "@hotwired/stimulus": "^3.2.2",
+    "@hotwired/turbo-rails": "^8.0.0-beta.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-redux": "^9.0.4",
+    "react-router-dom": "^6.20.1",
+    "redux": "^5.0.0",
+    "redux-thunk": "^3.1.0",
+    "webpack": "^5.89.0",
+    "webpack-cli": "^5.1.4"
+  },
+  "scripts": {
+    "build": "webpack --config webpack.config.js"
+  },
+  "devDependencies": {
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  }
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly you should also add `babel-loader` dependence inside of the `package.json` file as mentioned [this requirement](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/rails_react_webpack.md#:~:text=Update%20webpack.config.js%20to%20include%20the%20babel%20loader%20as%20described%20in%20the%20previous%20lesson%27s%20tutorial.) 🙏🏼